TileFormats/Instanced3DModel/README.md

```diff
@@ -22,61 +22,129 @@ A tile is composed of a header section immediately followed by a body section.
-| `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates there is not a batch table. |
+| `featureTableByteLength` | `uint32` | The length of the feature table in bytes. |
+| `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates that there is no batch table. |
 | `gltfByteLength` | `uint32` | The length of the glTF field in bytes. |
 | `gltfFormat` | `uint32` | Indicates the format of the glTF field of the body.  `0` indicates it is a url, `1` indicates it is embedded binary glTF.  See the glTF section below. |
-| `instancesLength` | `uint32` | The number of instances, also called features, in the tile. |
 
-_TODO: Link to Cesium code for reading header_
+If either `featureTableByteLength` or `gltfByteLength` equal zero, the tile does not need to be rendered.
 
-If either `gltfByteLength` or `instancesLength` equal zero, the tile does not need to be rendered.
+The body section immediately follows the header section, and is composed of three fields: `Feature Table`, `Batch Table`, and `glTF`.
 
-The body section immediately follows the header section, and is composed of three fields: `Batch Table`, `glTF`, and `instances`.
+Code for reading the header can be found in
+[Instanced3DModelTileContent](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Instanced3DModel3DTileContent.js#L170)
+in the Cesium implementation of 3D Tiles.
 
-## Batch Table
+## Feature Table
 
-_TODO: create a separate Batch Table spec that b3dm, i3dm, etc. can reference, [#32](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/32)?_
+Contains values for `i3dm` semantics used to create instanced models.
 
-The batch table is a `UTF-8` string containing JSON.  It immediately follows the header.  It can be extracted from the arraybuffer using the `TextDecoder` JavaScript API and transformed to a JavaScript object with `JSON.parse`.
+See the [Feature Table](TODO:add link) reference for more information.
 
-Each property in the object is an array with its length equal to `header.batchLength`.  Array elements can be any valid JSON data type, including objects and arrays.  Elements may be `null`.
+### Semantics
 
-An instance's `batchId` is used to access elements in each array and extract the corresponding properties.  For example, the following batch table has properties for two instances:
-```json
-{
-    "id" : ["unique id", "another unique id"],
-    "displayName" : ["Tree species", "Another tree species"],
-    "yearPlanted" : [1999, 2003],
-    "location" : [{"x" : 1, "y" : 2}, {"x" : 3, "y" : 4}]
-}
-```
+#### Instance Semantics
 
-The properties for the instance with `batchId = 0` are
-```javascript
-id[0] = 'unique id';
-displayName[0] = 'Tree species';
-yearBuilt[0] = 1999;
-location[0] = {x : 1, y : 2};
-```
+These semantics map to an array of feature values that are used to create instances. The length of these arrays must be the same for all semantics.
```

Comment:
Let's be explicit that length is number of instances, not byte length.

(in general this is what "length" means in 3D Tiles as opposed to "byteLength", but I think being extra explicit is good here).